projects/ccf-rui/src/app/shared/components/rotation-slider/rotation-slider.component.ts

```diff
@@ -65,8 +65,16 @@ export class RotationSliderComponent {
   /**
    * Function to easily reset the rotations to 0 and emit this change.
    */
-  resetRotation(): void {
-    this.rotation = DEFAULT_ROTATION;
+  resetRotation(slider: number): void {
```

Comment:
Rename slider input to dimension and change its type to `'x' | 'y' | 'z'`
Then you can simplify the update to `this.rotation = { ...this.rotation, [dimension]: 0 }`
It is also easier to see what is updated rather that having to remember that x <-> 1, y <-> 2, and z <-> 3